datadog_lambda/tag_object.py

```diff
@@ -0,0 +1,59 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+import json
+import logging
+
+redactable_keys = ["authorization", "x-authorization", "password", "token"]
```

Comment:
How did we decide on these four keys for redaction? Should we provide a way to redact other keys?

Reply:
We provide `datadog.yaml` obfuscation rules - these were 4 common items we wanted to automatically strip out to prevent accidental ingestion of secrets before a user has time to redact for themselves